tests/helper/helper_filesystem.go

```diff
@@ -108,3 +108,37 @@ func copyDir(src string, dst string, info os.FileInfo) error {
 	_, err = io.Copy(dFile, sFile)
 	return err
 }
+
+// CreateFileAtPathWithContent creates a file at the given path and writes the given content
+// path is the path to the required file
+// fileContent is the content to be written to the given file
+func CreateFileAtPathWithContent(path string, fileContent string) error {
```

Comment:
Some shorter name would be nicer.
Just `CreateFile` or if you really want `CreateFileWithContent`  might be better.